storage/shard.go

```diff
@@ -1019,7 +1039,9 @@ func (s *dbShard) FetchBlocks(
 	retriever := s.seriesBlockRetriever
 	onRetrieve := s.seriesOnRetrieveBlock
 	opts := s.seriesOpts
-	reader := series.NewReaderUsingRetriever(id, retriever, onRetrieve, opts)
+	// Nil for onRead callback because we don't want peer bootstrapping to impact
```

Comment:
super nit: maybe set a var to nil and pass that as the arg

Reply:
done